corehq/apps/accounting/models.py

```diff
@@ -3719,7 +3719,7 @@ def unset_autopay(self, card, billing_account):
 
     def _update_autopay_status(self, card, billing_account, autopay):
         stripe.Customer.modify_source(customer=self.customer.id, id=card.id,
-                                      metadata={self._auto_pay_card_metadata_key(billing_account): autopay})
+                                      metadata={self._auto_pay_card_metadata_key(billing_account): str(autopay)})
```

Comment:
I may not be following fully, but does `str(autopay)` need to be `str(autopay).lower()`?